### PR TITLE
Dev: cibconfig: "crm config show related:xxx" provides partial search among class, provider, type fields

### PR DESCRIPTION
**Pull request by liangxin1300**
```
crm(live/15sp4-1)configure# show related:ocf
primitive d Dummy
primitive vip IPaddr2 \
	params ip=10.10.10.125

crm(live/15sp4-1)configure# show related:stonith
primitive stonith-sbd stonith:external/sbd \
	params
primitive ttt stonith:rps10 \
	params pcmk_delay_max=30 serial_to_targets=sf
primitive xxx stonith:fence_eps \
	params pcmk_delay_base=10

crm(live/15sp4-1)configure# show related:ip
primitive vip IPaddr2 \
        params ip=10.10.10.125
```

### CODE REVIEW

File: crmsh/cibconfig.py [hunk 1]
```diff
@@ -3158,7 +3158,8 @@ def get_elems_on_ra_type(self, spec):
             tp = content_list[0]
             if not tp:
                 return []
-            return [x for x in self.cib_objects if x.node.get("type") == tp]
+            return [x for x in self.cib_objects if x.node.get("type") == tp or
+                    x.node.get("class") == tp]
```
Comment (zzhou1) [inline crmsh/cibconfig.py:3162]: Absolutely better UX! 

Further improvement, I prefer to do partial match to give more flexibility for human-being, something like,

```
tp in x.node.get("class") or tp in x.node.get("provider") or tp in x.node.get("type") 
```

The same for the other two situations: class:provider:type and class:type

Comment (zzhou1) [reply crmsh/cibconfig.py:3162]: Also, I would suggest a slightly different title for the git commit log, eg.

```
Dev: cibconfig: "crm config show related:xxx" provides partial search among class, provider, type fields
```

Comment (liangxin1300) [reply crmsh/cibconfig.py]: @zzhou1 
Thanks! Already changed!

File: crmsh/cibconfig.py [hunk 2]
```diff
@@ -3137,7 +3137,7 @@ def modified_elems(self):
 
     def get_elems_on_ra_type(self, spec):
```
Comment (zzhou1) [inline crmsh/cibconfig.py:3138]: Maybe rename the function name as well, something like,
```suggestion
    def get_elems_of_ra_partial_search(self, spec):
```

**Review by zzhou1 — APPROVED** [review]

**Pull request merged**